Solutions/Joshua-Cyberiskvision/Playbooks/Joshua-Indicators-Processor-DOMAIN/azuredeploy.json

```diff
@@ -45,7 +48,7 @@
      "LogicAppsCategory": "[parameters('LogicAppsCategory_Tag')]"
       },	
       "type": "Microsoft.Logic/workflows",
-      "apiVersion": "2016-06-01",
+      "apiVersion": "2019-05-01",
       "name": "[parameters('logicAppName')]",
```

Comment:
As given in the error in the pipeline validations, this name field must use the PlaybookName parameter. So "logicAppName" will need be changed to "PlaybookName" instead of "Playbook2Name" as that's the separate playbook. This change will need be made in each of the failing playbooks.